### DISCUSSION TITLE
PatchTool 6.1 4/22

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#152] [Announcements]
- 加入中國服6.01翻譯
- 修正大量簡繁轉換與錯別字
- 修正招募不會顯示所在位置的錯誤

檔案（兩者擇一）：
- FFXIVChnTextPatch-0422.rar：本體。一般而言，下載這個即可。JRE縮小過。
- FFXIVChnTextPatch-0422-full.rar：本體。JRE未縮小。有問題可以用用看這版。

選用：
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- rawexd-zhcn-0422.rar：簡體中文CSV。解壓後放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0415.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/backup-0415.rar)：乾淨的0x0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。
- [rawexd-ja-610-v2-0415.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/rawexd-ja-610-v2-0415.rar)：日文原文的CSV。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/'>PatchTool 6.1 4/22</a>.</em>

**Comment by z19881012** [comment]
感謝大大~~

**Comment by Manbbb** [comment]
感謝大大，辛苦了

**Comment by siler2011** [comment]
感謝大大!!

**Comment by Bally0886** [comment]
感謝大大!!

**Reply by KppKerry** [reply]
感謝好人

**Comment by sai750310** [comment]
在下載full版時Windows內建的防毒軟體Microsoft Defender跳出偵測到病毒，因為之前的版本下載都沒有碰過此種情形，不知道有沒有關係?
![image](https://user-images.githubusercontent.com/100848565/165006957-301a18ae-1a5a-42dd-9259-c2eecad4d8ae.png)
![image](https://user-images.githubusercontent.com/100848565/165006999-559b0b13-1e66-4d81-969f-5fc655e532e6.png)

**Reply by okami2223** [reply]
也有同樣問題，只有Full有這問題

**Comment by www1247700** [comment]
設定頁面有漢化
但是創角頁面跟開頭動畫還是日文

**Comment by biggou1593** [comment]
![123](https://user-images.githubusercontent.com/104715593/166140201-08e8727b-a35e-4ef6-9555-880153ce195b.png)
下載一打開就顯示這個，請問有什麼解決方法嗎

**Reply by tonyjediex** [reply]
直接選下面那個就可以了

**Reply by biggou1593** [reply]
解決了 謝謝

**Comment by Logan8246EX** [comment]
漢化後無法使用旅館的 旅行筆錄，會顯示"沒有查看旅行筆錄的許可權"，但我改用英文登入卻可以使用 沒問題
